### DISCUSSION TITLE
Quasar

### DISCUSSION THREAD

**Discussion by euphemism** [george-norton/headphones-toolbox#11] [General]
🤌 Someone linked the Ploopy Headphones site; I saw the screenshot, saw the Material UI, and thought to myself: "what if it were actually Quasar?" I hopped on over to the repo and checked `package.json`, and lo and behold: It is. Really great choice going with Vue + Quasar--I have been a very happy user for five years now. Noticing an interesting mix of JS/TS, and script setup + the options API. What's the reasoning behind these choices? Would love to help clean this up a bit and help where I can.

**Comment by george-norton** [comment]
Hello, I chose Quasar + Tauri because it's the same framework that the QMK project will be using for their future [XAP client](https://github.com/qmk/qmk_xap). I hoped to get something with a similar look and feel to the UI that will, in the future, be used to configure Ploopy's trackballs and mice.

The code is a bit of a mess of JavaScript and typescript as I am a firmware developer who quickly threw together a UI as best I could 😁 

I would happily accept a pull request if you are able to improve things.

**Reply by euphemism** [reply]
Nice. Will send you a PR at some point in the Near-ish™ future.
